src/ResourceManager/Compute/Commands.Compute/Manual/PSOperationStatusResponse.cs

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+using System;
+using Microsoft.Azure.Management.Compute.Models;
+
+namespace Microsoft.Azure.Commands.Compute.Automation.Models
+{
+    public class PSOperationStatusResponse
+    {
+        public string Name { get; set; }
```

Comment:
did this not exist before?  Are the pall the properties of the previous output type included?